Dockerfile.run

```diff
@@ -1,13 +1,17 @@
+FROM alpine:3.4
 
-FROM    alpine:3.4
-RUN     apk -U add \
-            python \
-            py-pip
 
-COPY    requirements.txt /code/requirements.txt
-RUN     pip install -r /code/requirements.txt
+ENV GLIBC 2.23-r3
 
-ADD     dist/docker-compose-release.tar.gz /code/docker-compose
-RUN     pip install --no-deps /code/docker-compose/docker-compose-*
 
-ENTRYPOINT ["/usr/bin/docker-compose"]
+RUN apk update && apk add --no-cache \
+    curl openssl ca-certificates \
+    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub \
+    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/$GLIBC/glibc-$GLIBC.apk \
```

Comment:
I missed this earlier. You're adding `curl`, but using `wget` to download. Why is that? Do we actually need both?